src/interface.tsx

```
@@ -74,10 +74,12 @@ export type KeyEntities<DateType extends BasicDataNode = any> = Record<
   DataEntity<DateType>
 >;
 
-export type DataNode = FieldDataNode<{
-  key: Key;
-  title?: React.ReactNode | ((data: DataNode) => React.ReactNode);
-}>;
+export type DataNode<T = {}> = FieldDataNode<
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)